CMakeLists.txt

```diff
@@ -19,22 +19,18 @@ set(NUKE_ROOT "/opt/nuke/Nuke11.3v4" CACHE PATH "Path to Nuke install root")
 set(CMAKE_CXX_COMPILER g++)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_FLAGS "-O3 -fPIC -msse -msse2 -msse3 -mssse3 -msse4 -msse4.1 -msse4.2 -mavx")
+set(CMAKE_CXX_FLAGS "-fPIC -msse -O3")
```

Comment:
Usually we avoid to specify explicit flags: CMake has better, more portable cross-compilers functions
https://stackoverflow.com/questions/38296756/what-is-the-idiomatic-way-in-cmake-to-add-the-fpic-compiler-option